server/api/include/irods/rs_replica_truncate.hpp

```diff
@@ -0,0 +1,88 @@
+#ifndef IRODS_RS_REPLICA_TRUNCATE_HPP
+#define IRODS_RS_REPLICA_TRUNCATE_HPP
+
```

Comment:
Likely need the `/// \file` directive here as well.

Reply:
Done